pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -65,7 +65,7 @@ in
 , broken ? false
 , preCompileBuildDriver ? null, postCompileBuildDriver ? null
 , preUnpack ? null, postUnpack ? null
-, patches ? null, patchPhase ? null, prePatch ? "", postPatch ? ""
+, patches ? null, postPatch ? null, prePatch ? "", postPatch ? ""
```

Comment:
What are you trying to do here?  This doesn't make sense and will likely not evaluate.